config/groups.yaml

```
@@ -551,3 +551,9 @@
     - "2428040"
     - "2475521"
   id: 190025
+  - name: "RKHW"
+  members:
+    -"2598057"
+    -"2614594"
+    -"2564504"
+    -"2641881"
```

This isn't valid YAML. 
```suggestion
- name: "RKHW"
  members:
    - "2598057"
    - "2614594"
    - "2564504"
    - "2641881"
```

---

This is still not fixed